ui/page/send/send_amount.go

```diff
@@ -21,9 +21,9 @@ type sendAmount struct {
 	dcrAmountEditor decredmaterial.Editor
 	usdAmountEditor decredmaterial.Editor
 
-	sendMax               bool
-	dcrSendMaxChangeEvent bool
-	usdSendMaxChangeEvent bool
+	Sendmax               bool
```

Comment:
```suggestion
	SendMax               bool
```
replace all instances